### DISCUSSION TITLE
Las Vegas Race

### DISCUSSION THREAD

**Discussion by McHouse** [theOehrly/Fast-F1#486] [Q&A] [answered]
I don't know, but, am I the only one for whom this doesn't work?
`
import fastf1
Session = fastf1.get_session(2023, 'Las Vegas', 'R')
Session.load()
Resultados=Session.results
Laps=Session.laps
`
exit:
Laps=Session.laps

  File ~\anaconda3\envs\FF1_V3\lib\site-packages\fastf1\core.py:1105 in laps
    return self._get_property_warn_not_loaded('_laps')

  File ~\anaconda3\envs\FF1_V3\lib\site-packages\fastf1\core.py:1066 in _get_property_warn_not_loaded
    raise DataNotLoadedError("The data you are trying to access has not "

DataNotLoadedError: The data you are trying to access has not been loaded yet. See `Session.load`

Thanks in advance

**Comment by theOehrly** [comment, marked as answer]
Have you updated to the latest version? (see #481)

**Reply by McHouse** [reply]
🤦  No words.
Thanks theOehrly 🥇
